ui/src/app/cohort-search/modifier-page/modifier-page.component.ts

```diff
@@ -110,7 +119,18 @@ export class ModifierPageComponent implements OnInit, OnDestroy, AfterContentChe
 
   dateA = new FormControl();
   dateB = new FormControl();
-  showError = false;
+  errors = new Set();
+  readonly errorMessages = {
+    ageAtEvent: {
+      range: 'Values for Age At Event must be between 1 and 120',
+      integer: 'Values for Age At Event must be valid integers'
+    },
+    hasOccurrences: {
+      range: 'Values for Has Occurrences must be between 1 and 99',
+      integer: 'Values for Has Occurrencest must be valid integers'
```

Comment:
typo: please remove the t from Occurrencest

Reply:
Done